--- cacti-addons-0.1/cacti/add_template.php.orig	2005-08-09 12:28:48.000000000 +0000
+++ cacti-addons-0.1/cacti/add_template.php	2007-12-25 04:37:28.000000000 +0000
@@ -7,9 +7,8 @@
 | Version : 0.1 8 Aug 2005						    |
 +---------------------------------------------------------------------------+
 */
-
-include_once("./include/config.php");
-include_once("./lib/import.php");
+include_once("/usr/share/cacti/include/config.php");
+include_once("/usr/share/cacti/lib/import.php"); 
 
 // check if we have good number of argument
 if ($argc != 2 || in_array($argv[1], array('--help', '-help', '-h', '-?')))
@@ -30,7 +29,7 @@
 $xml_data = fread($fp,filesize($xml_file));
 
 // import data into cacti. $info_array will contain debugging info.
-$info_array = import_xml_data($xml_data);
+$info_array = import_xml_data($xml_data,false);
 
 return 0;
 ?>
